packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -113,29 +114,54 @@ export function mergeUi5(ui5: Partial<UI5>): UI5 {
 
     merged.descriptorVersion = getManifestVersion(merged.minUI5Version, ui5.descriptorVersion);
     merged.typesVersion = ui5.typesVersion ?? getTypesVersion(merged.minUI5Version);
+    merged.esmTypesVersion = ui5.esmTypesVersion ?? getEsmTypesVersion(merged.minUI5Version);
     merged.ui5Theme = ui5.ui5Theme ?? 'sap_fiori_3';
     merged.ui5Libs = getUI5Libs(ui5.ui5Libs);
 
     return Object.assign({}, ui5, merged) as UI5;
 }
 
 /**
- * Get the best types version for the given minUI5Version.
+ * Get the best types version for the given minUI5Version for https://www.npmjs.com/package/@sapui5/ts-types where specific versions are missing
  * For the latest versions the LTS S/4 on-premise version (1.102.x) is used, for anything before we match the versions as far back as available.
  *
```

Comment:
```suggestion
```
this relates to the esm module not the one for JS.

Reply:
Starting with `1.90` both are always generated whenever a new UI5 version is generated. @petermuessig correct me if I am stating something wrongly here.